added using Substitute as an option

syto203 opened the pull request.

syto203 committed:
added using Substitute as an option

Randomblock1 commented:
This works really well, but I've found an issue with it. When using ./install to download Substitute, switching to Substrate, and then back to Substitute, it won't put Cydia Substrate Framework in the IPA's Frameworks. In modules/lib/, Cydia Substrate Framework is named CydiaSubstrate_0.9.6301.framework instead of just CydiaSubstrate.framework, so something is not working right in the install script.

I think this is because the install script starts _permanently_ removing files around [here](https://github.com/kabiroberai/theos-jailed/blob/299cce6f9d8acc4e9727e100fd665e6322630391/install#L73).
It would be much better if it used symbolic links instead to preserve every file. For example, instead of ipa_substitute.sh completely replacing and deleting ipa.sh, ipa.sh is just a symbolic link to either the new ipa script or the old one (which would be renamed to ipa_substrate.sh). This way, switching between the two doesn't delete any files, just changes links.

That being said, `git reset --hard` to reset the repository and then just running `./install` again fixes it.

syto203 commented:
Sorry I was half asleep when I submitted this hence the multiple pushes. I didn’t take into account someone juggling between both but the fix will be to remove the Cydiasubstrate.framework and download and use the new one. I’ll modify the script soon.

syto203 committed:
Fix: fixed shuffling between both substrate and substitute

also removed downloaded binaries so you always update to latest if you run the installer.

Signed-off-by: syto203 <26351342+syto203@users.noreply.github.com>
